packages/components/src/drop-zone/index.tsx

```diff
@@ -116,8 +117,6 @@ export function DropZoneComponent( {
 			( ( type === 'file' && onFilesDrop ) ||
 				( type === 'html' && onHTMLDrop ) ||
 				( type === 'default' && onDrop ) ),
-		'has-dragged-out': ! isDraggingOverElement,
-		// Keeping the following classnames for legacy purposes
```

Comment:
The `.has-dragged-out` class was added in #62044, but it seems to work fine with `.is-dragging-over-document` that existed prior to #62044. Maybe @tyxla or @ciampo know something more?

I assume the classes that existed before were utility classes for consumers to add additional styling, so I removed the comment too.

It should be safe to remove as it's a recent change for 6.7 and isn't used anywhere else.